LayoutTests/imported/w3c/web-platform-tests/css/css-animations/animation-name-in-shadow-part-inner-match.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Matching animation-name from within a shadow root with multiple @keyframes rules</title>
```

Comment:
These tests are not as comprehensive as css/css-anchor-position/position-try-tree-scoped.html which also tests nested shadow trees.

Reply:
I added a fourth test which show nesting similar to `css/css-anchor-position/position-try-tree-scoped.html`.